Report.md

```diff
@@ -888,8 +888,8 @@ an if/elseif statement to execute later blocks and perform more comparisons.
 Second, the random inputs may cause more cache misses, as the program
 alternates more between reading new elements from the two input arrays.
 
-![alt text](merge_ipynb/Merge_Plots/Mergesort%20Strong%20Scaling%20(comp_large,%20n=268435456).png)
-![alt text](merge_ipynb/Merge_Plots/Mergesort%20Strong%20Scaling%20Speedup%20(comp_large,%20Random).png)
+![alt text](radix_ipynb/Radix_Plots/Radix%20Strong%20Scaling%20(comp_large,%20n=268435456).png)
+![alt text](radix_ipynb/Radix_Plots/Radix%20Strong%20Scaling%20Speedup%20(comp_large,%20Random).png)
```

Comment:
This is part of the Merge Sort Computation section. These shouldn't be Radix Sort graphs.

Reply:
Got it, it should be resolved now.